### ISSUE TITLE
on instance mof output we do not properly fold the first line of a long string value.

### ISSUE THREAD

**Issue by KSchopmeyer** [pywbem/pywbem#343]
```
instance of PG_ComputerSystem {
    Status = "OK";
    ElementName = "Computer System";
    PowerState = 1;
    Description = "Linux version 3.13.0-86-generic (buildd@lgw01-51) (gcc versio
n 4.8.2 "
    "(Ubuntu 4.8.2-19ubuntu1) ) #131-Ubuntu SMP Thu May 12 23:33:13 UTC 2016";
    PowerManagementSupported = False;
```

**Comment by andy-maier** [comment]
I thought that MOF does not allow that literal strings spill over into new lines and that they must be double quoted on each line separately. In other words, I thought that U+000A must be escaped. 

However, after re-reading section .12.1 "String Constants" in DSP0004 2.8.0, it turns out that is not stated there. So I assume having U+000A in a string constant is legal MOF.

If we agree on that, this issue is a normal bug, but no standards compliance violation.

**Comment by KSchopmeyer** [comment]
There is at least one more case where we do not fold correctly in mof output, the qualifier description last line.

Example: 

class CIM_MethodParameters : CIM_ManagedElement {

        [Description ("The identifier by which the MethodParameters object is known."),
        MaxLen (256)]
    string MethodParametersId;

the 'known."),' exceeds 80 characters.

It appears that the issue is tied to the last line of a description when the last word of that line exceeds 80 char or the addition of the ")," drives it over the 80 character length.

**Comment by andy-maier** [comment]
Fixed in PR #942.
